docs/DEVELOPMENT_NOTES.md

```diff
@@ -48,6 +52,17 @@ This is the same as above, only all javascript files are published via IPFS.
 This more closely emulates a production environment, but any frontend or worker script changes will
 require a republish to view changes in the browser.
 
+#### To run a specific app for development purposes
+
+The following commands will create a DAO with the Allocations app, and the apps it depends on, namely Dot Voting and Address Book.
+
+- `npm run devchain:reset` : Start a local blockchain
+- `npm run dev:allocations` : Start the development server (instead of IPFS)
+- `npm run publish:allocations` : Publish the contracts on the local blockchain
+- `npm run start:allocations` : Set up an Aragon DAO with the selected apps
+
+Note that you need to run each of these commands (except publish) on separate windows.
```

Comment:
If these could all be run with a single command from the project root (`npm run start:allocations` or `npm run dev:allocations` maybe), then I think it might be worth adding this change. 

How much time do you think using this script, vs using `start:dev`, would save?

Reply:
I think it is not matter of "how much do you think". We should try to benchmark these things to have better metrics on the potential benefits.